.devcontainer/devcontainer.json

```diff
@@ -11,11 +11,10 @@
                 "ms-python.python",
                 "ms-python.vscode-pylance",
                 "ms-python.black-formatter",
-                "GitHub.vscode-pull-request-github",

```

Comment:
What was the purpose of removing this dependency?

Reply:
Its just an integration with GitHub for pull requests. I never used it, I'm not sure if you all have. Since it's not directly related to our project so I assume anyone wanting to use it can just install it locally.

Reply:
Ah makes sense